src/styles/stars/stars.css

```diff
@@ -0,0 +1,39 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars__star {
+  display: inline;
+  margin: 0 2px;
+  background-position: center;
+}
+
+.stars__text {
+  display: flex;
+  align-items: baseline;
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+  margin-right: 16px;

```

Comment:
Why do you need to add the same margin to each section? Just add common padding to the card 
<img width="287" alt="image" src="https://user-images.githubusercontent.com/99403198/218100608-371e1098-ebc3-46a0-8e94-60cbe08b3a9a.png">